package.json

```diff
@@ -8,10 +8,37 @@
     "node": ">=6.13.0"
   },
   "scripts": {
+    "build": "webpack -d --watch",
+    "start": "node ./server/server.js",
+    "devserver": "nodemon ./server/server.js",
+    "test": "jest"
+  },
+  "jest": {
+    "setupFilesAfterEnv": [
+      "<rootDir>client/setupTests.js"
+    ]
   },
   "dependencies": {
+    "@babel/core": "^7.5.0",
+    "@babel/polyfill": "^7.4.4",
+    "@babel/preset-env": "^7.5.0",
+    "@babel/preset-react": "^7.0.0",
+    "babel-core": "^6.26.3",
+    "babel-loader": "^8.0.6",
+    "enzyme": "^3.10.0",
+    "enzyme-adapter-react-16": "^1.14.0",
+    "express": "^4.17.1",
+    "jest": "^24.8.0",
+    "path": "^0.12.7",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    "react-modal": "^3.8.2",
```

Comment:
You should look into creating modals from scratch if it's a critical aspect of your module rather than using something prebuilt.